### PR TITLE
Bug/logout

### PR DESCRIPTION
**Pull request by bgmad**
## Bug fix: redirects user to '/welcome' if there is no token

**Description:** 
changed some routes to private routes and made changes to `privateRoute.js` to make it functional

### CODE REVIEW

File: src/assets/Dashboard.js [hunk 1]
```diff
@@ -55,15 +55,15 @@ const Dashboard = (props) => {
...
                   component={RoomContent}
-                ></Route>
-                <Route
+                ></PrivateRoute>
+                <PrivateRoute
                   exact
                   path={`${path}/`}
                   component={DashboardContent}
-                ></Route>
+                ></PrivateRoute>
```
Comment (mxhs) [inline src/assets/Dashboard.js]: Can all of the above PrivateRoutes be self closing tags? Would be cleaner.

File: src/assets/DiscussionCard.js [hunk 2]
```diff
@@ -66,10 +68,10 @@ const DiscussionCard = (props) => {
         <FlagManagerModal visible={showModal} setVisible={setShowModal} />
       </Card>
       <Switch>
-        <Route
+        <PrivateRoute
           path={`${path}/discussion/:discussionID`}
           component={DiscussionDrawer}
-        ></Route>
+        ></PrivateRoute>
```
Comment (mxhs) [inline src/assets/DiscussionCard.js]: Self closing again? `<PrivateRoute ... />`

**Review by mxhs — CHANGES_REQUESTED** [review]
Solid! Just would love to see the Private Routes having self closing tags to reduce clutter.

File: src/assets/Dashboard.js (hunk 1 above)
Comment (bgmad) [reply src/assets/Dashboard.js]: No problem all done

**Review by mxhs — APPROVED** [review]

**Review by CPower1248 — APPROVED** [review]

**Pull request merged**